template.yaml

```diff
@@ -91,11 +120,40 @@ objects:
                 value: spark://${MASTER_NAME}:7077
               - name: SPARK_MASTER_UI_ADDRESS
                 value: http://${MASTER_NAME}-webui:8080
-
+              - name: NSS_WRAPPER_PASSWD
+                value: /tmp/passwd
+              - name: NSS_WRAPPER_GROUP
+                value: /etc/group
+              - name: LD_PRELOAD
+                value: libnss_wrapper.so
+            volumeMounts:
+              - mountPath: /data
+                name: spark-data
+        volumes:
+          - name: spark-data
+            persistentVolumeClaim:
+              claimName: spark-claim
+- apiVersion: v1
+  kind: Route
+  metadata:
+    annotations:
+      openshift.io/host.generated: "true"
+    creationTimestamp: null
+    labels:
+      app: apachespark-${MASTER_NAME}-webui
+    name: ${MASTER_NAME}
+  spec:
+    port:
+      targetPort: 8080
+    to:
+      kind: Service
+      name: ${MASTER_NAME}-webui
+  status:
+    ingress: null
 parameters:
 - name: SPARK_IMAGE
   description: Name of the Spark master/worker image
-  value: _REPO_/openshift-spark:latest
+  value: 10.193.127.18:5000/openshift/apachespark:latest
```

Comment:
since this is the default value for the parameter, i think it makes the most sense to have it as `docker.io/radanalyticsio/openshift-spark:latest` to ensure that our users will default to our known and tested image.